location/src/main/kotlin/io/rover/campaigns/location/GoogleBackgroundLocationService.kt

```diff
@@ -126,7 +160,6 @@ class GoogleBackgroundLocationService(
                         .create()
                         .setInterval(LOCATION_UPDATE_INTERVAL)
                         .setFastestInterval(LOCATION_UPDATE_INTERVAL)
-                        .setSmallestDisplacement(minimumDisplacement)
```

Comment:
consider restoring this so google has opportunity to cut down on unnecessary updates from their side. initial update will still happen!